examples/iam-access-groups.v2.test.js

```diff
@@ -664,4 +692,727 @@ describe('IamAccessGroupsV2', () => {
+    // begin-update_template_version
+
+    // Request models needed by this operation.
+
+    // MembersActionControls
+    const membersActionControlsModel = {
+      add: true,
+      remove: false,
+    };
+
+    // MembersInput
+    const membersInputModel = {
+      users: ['IBMid-665000T8WY'],
+      action_controls: membersActionControlsModel,
+    };
+
+    // ConditionInput
+    const conditionInputModel = {
+      claim: 'blueGroup',
+      operator: 'CONTAINS',
+      value: '\"test-bluegroup-saml\"',
+    };
+
+    // RulesActionControls
+    const rulesActionControlsModel = {
+      remove: false,
+      update: false,
+    };
+
+    // RuleInput
+    const ruleInputModel = {
+      name: 'Manager group rule',
+      expiration: 12,
+      realm_name: 'https://idp.example.org/SAML2',
+      conditions: [conditionInputModel],
+      action_controls: rulesActionControlsModel,
+    };
+
+    // AssertionsActionControls
+    const assertionsActionControlsModel = {
+      add: false,
+    };
+
+    // AssertionsInput
+    const assertionsInputModel = {
+      rules: [ruleInputModel],
+      action_controls: assertionsActionControlsModel,
+    };
+
+    // AccessActionControls
+    const accessActionControlsModel = {
+      add: false,
+    };
+
+    // GroupActionControls
+    const groupActionControlsModel = {
+      access: accessActionControlsModel,
+    };
+
+    // AccessGroupInput
+    const accessGroupInputModel = {
+      name: 'IAM Admin Group 8',
+      description: 'This access group template allows admin access to all IAM platform services in the account.',
+      members: membersInputModel,
+      assertions: assertionsInputModel,
+      action_controls: groupActionControlsModel,
+    };
+
+    // PolicyTemplatesInput
+    const policyTemplatesInputModel = {
+      id: testPolicyTemplateId,
+      version: '1',
+    };
+
+    const params = {
+      templateId: testTemplateId,
+      versionNum: '1',
+      ifMatch: testTemplateEtag,
+      name: 'IAM Admin Group template 2',
+      description: 'This access group template allows admin access to all IAM platform services in the account.',
+      group: accessGroupInputModel,
+      policyTemplateReferences: [policyTemplatesInputModel],
+      transactionId: '83adf5bd-de790caa3',
+    };
+
+    let res;
+    try {
+      res = await iamAccessGroupsService.updateTemplateVersion(params);
+      console.log(JSON.stringify(res.result, null, 2));
+    } catch (err) {
+      console.warn(err);
+    }
+
+    // end-update_template_version
+  });
+
+  test('getTemplateVersion request example 1', async () => {
+    consoleLogMock.mockImplementation((output) => {
+      originalLog(output);
+    });
+    consoleWarnMock.mockImplementation((output) => {
+      // if an error occurs, display the message and then fail the test
+      originalWarn(output);
+      expect(true).toBeFalsy();
+    });
+
+    originalLog('getTemplateVersion() result:');
+    // begin-get_template_version
```

Comment:
Limited to only 1 code example per operation.

Reply:
fixed, and test results updated.